platform/integrations/slack_integration/usage.mdx

```diff
@@ -0,0 +1,104 @@
+---
+id: usage
+title: Usage
+sidebar_label: Usage
+description: Learn how to use commands, receive notifications about Tolgee project updates in Slack, customize the notifications, and more.
+---
+
+import { FigmaScreenshotWrapper } from '../../shared/_FigmaScreenshotWrapper';
+
+# Usage
+
+This guide will help you subscribe to project updates, manage your notifications, and more. Whether you're a project manager, developer, or translator,
+ Tolgee's Slack integration offers a seamless way to streamline your workflow and collaborate more efficiently with your team. :rocket:
+
+## Subscribe to a Project
+
+Subscribe to a project to receive notifications for all events and languages in the project.
+
+``
+/tolgee subscribe $project_id
+``
+
+After subscribing, you will receive notifications on key creation, and translation update events.
+
+<FigmaScreenshotWrapper src="/img/docs/platform/slack_integration/global_subscription.png" />
+
+## Advanced Subscription Options
+
+### Language Subscription
+
+You can subscribe to notifications for a specific language within a project. This allows you to receive updates only
```

Comment:
is this true? It also subscribes you to the new keys creation and base language updates, right? Also there is no mentions about the concept of posting new messages when subscribed to specific language.